tests/chainer_tests/distributions_tests/test_bernoulli.py

```diff
@@ -156,5 +163,16 @@ def test_double_backward_gpu(self):
             cuda.to_gpu(self.logit), cuda.to_gpu(self.x),
             cuda.to_gpu(self.gy), cuda.to_gpu(self.ggx))
 
+    def test_backward_where_logit_has_infinite_values(self):
+        self.logit += numpy.inf
```

Comment:
For the readability,
```suggestion
        self.logit[...] = numpy.inf
```
looks better to me.

Reply:
Thank you.
I'll fix it.

Reply:
I fixed it.